resources/assets/coffee/osu_common.coffee

```diff
@@ -242,11 +235,6 @@
   transChoice: (key, count, replacements) ->
     message = Lang.choice key, count, replacements, currentLocale
```

Comment:
That's the original, there's no point changing it since we can just get rid of it